.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: python
+python:
+    - "2.7"
+    - "2.6"
+before_install:
+    - git config --global user.email "OpenStack_TravisCI@f5.com"
+    - git config --global user.name "Travis F5 Openstack"
+install:
+    - pip install requests hacking pytest pytest-cov
+script:
+    - flake8 .
+    - py.test --cov --ignore=test/
+notifications:
+    slack:
+      rooms:
+        - f5openstackdev:$SLACK_PROJECT_TOKEN
```

Comment:
The only thing I'm unsure about here is where the project / build status tokens have to be set up. I believe @pjbreaux or @swormke should know.